src/build/arg/mod.rs

```diff
@@ -4537,6 +4622,7 @@ impl<'help> From<&'help Yaml> for Arg<'help> {
                 "required" => yaml_to_bool!(a, v, required),
                 "required_if_eq" => yaml_tuple2!(a, v, required_if_eq),
                 "required_if_eq_any" => yaml_tuple2!(a, v, required_if_eq),
+                "required_if_eq_all" => yaml_array_tuple2!(a, v, required_if_eq_all),
```

Comment:
Not sure why you need this separate function.

Reply:
Oh I understand now. Please change the above line too.

Reply:
because `required_if_eq_any` is using `required_if_eq`, which takes the key and value arguments instead of an array.